erreur/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "erreur"
+version = "0.1.0"
+authors = ["Pascal Hertleif <pascal@technocreatives.com>"]
+edition = "2018"
+
+[dependencies]
+eyre = "0.4"
+color-eyre = "0.2"
```

Comment:
It doesn't look like you're using tracing so you might want to use `default-features = false` to disable the tracing deps